rasa/shared/exceptions.py

```diff
@@ -111,3 +111,7 @@ class ConnectionException(RasaException):
     It's used by our broker and tracker store classes, when
     they can't connect to services like postgres, dynamoDB, mongo.
     """
+
+
+class TFLayerConfigException(RasaException):
```

Comment:
@dakshvar22 Can we move this to `rasa.utils.tensorflow.layers`? There is no need to have this in the shared package.

Reply:
Should we have an `exceptions.py` inside `rasa.utils.tensorflow` and place it inside that?

Reply:
up to you 👍🏻